src/main/java/com/frc2879/newcomen/RobotModule.java

```diff
@@ -109,6 +111,7 @@ public void teleopInit() {
         // continue until interrupted by another command, remove
         // this line or comment it out.
         if (autonomousCommand != null) autonomousCommand.cancel();
+        new DriveMecanum().start();
```

Comment:
You don't want to do this. The correct way is to set DriveMecanum as the default command of the DriveTrain subsystem.